apps/web/package.json

```diff
@@ -3,9 +3,9 @@
   "private": true,
   "version": "0.0.0",
   "scripts": {
-    "dev": "TAMAGUI_TARGET='web' vite",
-    "build": "tsc && vite build",
-    "preview": "vite preview",
+    "dev": "TAMAGUI_TARGET=web vite",
+    "build": "tsc && TAMAGUI_TARGET=web vite build",
+    "preview": "TAMAGUI_TARGET=web vite preview",
```

Comment:
otherwise `Must set TAMAGUI_TARGET` error. not even setting process.env in the config as the first line or passing it to `define` would not work

Reply:
ideally the vite would use `.env.*` files at least for itself as first thing before bundling